src/style.css

```
@@ -1 +1,72 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src:
+    local("Avenir"),
+    url("fonts/Avenir-Book.ttf") format("truetype"),
+    url("fonts/Avenir-Heavy.ttf") format("truetype"),
+    url("fonts/Avenir.ttc") format("collection");
+}
+
+.bars {
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  box-sizing:border-box;
```

You can move box sizing to the * selector
```suggestion
  box-sizing: border-box;
```